pyviz3d/visualizer.py

```diff
@@ -81,7 +92,7 @@ def add_bounding_box(self, name, position, size, orientation=None):
         """
         self.elements[name] = Cuboid(position, size, orientation)
 
-    def save(self, path, port=6008):
+    def save(self, path, port=6008, verbose=True):
```

Comment:
Should we put the default on False?

Reply:
It would be more useful, but it might confuse new users.
I made it in the way that the default behaviour is not changed.

Reply:
IMHO removing `print` statements and writing how to run server in the docs is the best option.